frontend/src/app/lib/addon.js

```diff
@@ -46,8 +46,9 @@ export function installAddon(store, eventCategory, experimentTitle) {
   };
 
   if (useMozAddonManager) {
-    sendToGA('event', gaEvent);
     mozAddonManagerInstall(downloadUrl).then(() => {
+      gaEvent.eventValue = RESTART_NEEDED ? 1 : 0;
```

Comment:
Yeah, but when you're giving numbers magic values, I'd rather be explicit about what the range of values are.